tqdm/_tqdm.py

```diff
@@ -944,8 +945,10 @@ def set_postfix(self, **kwargs):
             # Else for any other type, try to get the string conversion
             elif not isinstance(kwargs[key], basestring):
                 kwargs[key] = str(kwargs[key])
+            # Else if it's a string, don't need to preprocess anything
```

Comment:
@lrq3000 yes, but should it be moved to before... wait the code changed... but it didn't... I don't know what drug I was on when I reviewed that. Damn.